packages/launcher/src/renderer/launcher/wallets/WalletAddLedgerModal.js

```diff
@@ -91,28 +102,38 @@ export default class WalletAddLedgerModal extends Component<Props, State> {
     ledgerName: '',
     hover: '',
     saving: false,
+    hdPath: LEDGER_LEGACY_PATH,
   }
 
   componentDidMount() {
     this.fetchAccounts()
   }
 
   onChangePage = (pageNumber: number) => {
-    this.fetchAccounts(pageNumber)
+    this.setState({ currentPage: pageNumber }, () => {
+      this.fetchAccounts()
```

Comment:
I don't think it's necessary to do this call in the `setState()` callback as `fetchAccounts()` does not mutate the same field, it could be simply:
```js
this.setState({ currentPage: pageNumber })
this.fetchAccounts()
```